### DISCUSSION TITLE
无法触发机器人，是需要添加机器人微信吗

### DISCUSSION THREAD

**Discussion by ayaweiy** [zhayujie/chatgpt-on-wechat#133] [Q&A]
现在微信已经添加不了您了，请问微信扫码后无法触发，检查config也没有出现问题，这边如何解决呢？个人聊天中触发，是说随便一个聊天窗口都可以触发吗？还是要添加某个微信号？

**Comment by ayaweiy** [comment]
> 现在微信已经添加不了您了，请问微信扫码后无法触发，检查config也没有出现问题，这边如何解决呢？个人聊天中触发，是说随便一个聊天窗口都可以触发吗？还是要添加某个微信号？
已经成功了，是登录超时了，log里面有。我想请问是不是运行成功了就一直可以用？还是必须保持app.py运行呢？

**Comment by zhayujie** [comment]
扫码后你就是机器人了，随便一个聊天窗口即可触发 (你找好友或好友找你)
